packages/ui/src/data/sqon/operators.ts

```diff
@@ -0,0 +1,16 @@
+export const GT_OP = '>';
+export const LT_OP = '<';
+export const BETWEEN_OP = 'between';
+export const GTE_OP = '>=';
+export const LTE_OP = '<=';
+export const IN_OP = 'in';
+export const NOT_IN_OP = 'not-in';
+export const ALL_OP = 'all';
+export const FIELD_OPS = [GT_OP, LT_OP, BETWEEN_OP, GTE_OP, LTE_OP, IN_OP, NOT_IN_OP, ALL_OP];
+export const RANGE_OPS = [GT_OP, LT_OP, BETWEEN_OP, GTE_OP, LTE_OP];
+export const TERM_OPS = [IN_OP, ALL_OP, NOT_IN_OP];
+
+export const AND_OP = 'and';
+export const OR_OP = 'or';
+export const NOT_OP = 'not';
+export const BOOLEAN_OPS = [AND_OP, OR_OP, NOT_OP];
```

Comment:
I think it would be clearer if regroup in either in JS object (ts also) or ts enums. It would also eased the type definifion and import later on.

With typescript enum you can use either `includes` or `in`.